bundles/deployment_admin/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-celix_subproject(DEPLOYMENT_ADMIN "Option to enable building the Deployment Admin Service bundles" ON DEPS framework launcher shell_tui log_writer)
```

Comment:
Framework is always built. shell_tui and log_writer are only used for the demo application, thus are of non-essential usage.